platform/ios/jazzy.yml

```diff
@@ -17,9 +17,6 @@ framework_root: ../darwin/src
 custom_categories:
   - name: Guides
```

Comment:
Should we keep calling this section “Guides” or refashion it as “Appendices” to clarify their scope?